src/systemctl/systemctl.c

```diff
@@ -145,7 +145,6 @@ static char *arg_root = NULL;
 static usec_t arg_when = 0;
 static char *argv_cmdline = NULL;
 static enum action {
-        _ACTION_INVALID,
```

Comment:
with this change, verb_to_action()'s loop needs to be updated and maybe start explicitly from 0